### DISCUSSION TITLE
Rendering XAML content to a transparent native window

### DISCUSSION THREAD

**Discussion by sudongg** [AvaloniaUI/Avalonia#7335] [General]
I know it is possible to render skia -SKCanvas to a transparent native window (created from CreateWindowExW).
Does Avalonia provide a way to render XAML content to a transparent native window?

**Comment by nielsbosma** [comment]
Yes, set this on the Window:
HasSystemDecorations=False
Background=Transparent

**Reply by sudongg** [reply]
Instead of rendering to Avalonia's Window, it's a transparent native window created using the Win API CreateWindowExW.
For some reason I can't use Avalonia's Window directly.
